README.md

```diff
@@ -6,6 +6,8 @@
 > Portfolio project
 > Add contact Session project
 > Add formspree id link
+> Add Animation effect to the project
+> Make the Desktop version of the protfolio project
 
 ## Built With
```

Comment:
- [ ] You should update your ReadMe  as per the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. For example, `Getting Started` (Prerequisites, Setup, Install, Usage, Run tests, Deployment), `Contributing`, `Show your support` and `Acknowledgments` parts should be added.
- [ ] Kindly remove the placeholder `Project Name` text in the ReadMe
![Readme-project-name](https://user-images.githubusercontent.com/44057863/167258802-6212e090-bc44-4f6b-8462-6fd73a9f42b5.png)
.

Reply:
thanks 
I removed the project name heading and I added what you request i do